logstash_async/formatter.py

```diff
@@ -202,7 +202,11 @@ def _get_extra_fields(self, record):
         }
         # static extra fields
         if self._extra:
-            extra_fields.update(self._extra)
+            for field_name, field_value in self._extra.items():
+                if callable(field_value):
+                    field_value = field_value()
```

Comment:
I wonder if we should catch errors from the callable (and if so, move it into a seperate method).

If the callable throws an error, this way the whole log event will be lost.
If the error is handled, the log event can be sent anyways and we could log the error for later analysis.

What do you think?

Reply:
@piotrmaslanka what do you think?